### PR TITLE
enable qe-catalogsource

### PR DESCRIPTION
**Pull request by Xia-Zhao-rh**
@kuiwang02

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/cucushift-installer-rehearse-vsphere-upi-workflow.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ workflow:
     - ref: upi-conf-vsphere-ova
     - ref: upi-install-vsphere
     - ref: ipi-install-times-collection
+    - ref: enable-qe-catalogsource
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/cucushift-installer-rehearse-vsphere-upi-workflow.yaml:13]: I think we can replace all the pre steps with `cucushift-installer-rehearse-vsphere-upi-provision` chain, and all the step with `cucushift-installer-rehearse-vsphere-upi-deprovision` chain.

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/proxy/https/cucushift-installer-rehearse-vsphere-upi-proxy-https-workflow.yaml [hunk 2]
```diff
@@ -11,6 +11,7 @@ workflow:
     - ref: upi-conf-vsphere-ova
     - ref: upi-install-vsphere
     - ref: ipi-install-times-collection
+    - ref: enable-qe-catalogsource
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/proxy/https/cucushift-installer-rehearse-vsphere-upi-proxy-https-workflow.yaml:14]: I think we can replace all the pre steps with `cucushift-installer-rehearse-vsphere-upi-proxy-https-provision` chain, and all the post steps with `cucushift-installer-rehearse-vsphere-upi-proxy-https-deprovision` chain.

**Comment by Xia-Zhao-rh** [comment]
@jianlinliu Please help to review it, thanks

**Comment by Xia-Zhao-rh** [comment]
@jianlinliu Please help to review it, thanks

**Pull request merged**